### DISCUSSION TITLE
嫌桌面太無聊? 來試試這款桌面美化軟體Wallpaper Engine吧!

### DISCUSSION THREAD

**Discussion by a45320614** [1101-GS4719-C/Python-Community-GS4719-C#685] [General]
今天要來分享的是一款叫做Wallpaper Engine的桌面美化軟體

![image](https://user-images.githubusercontent.com/92359154/191426943-8664587c-580e-48e7-9db0-006df62fbb55.png)

Wallpaper Engine是一款在Steam上販售的桌面美化軟體
它提供了數以萬計的桌布，不論是動漫、遊戲、大自然、動物、像素等等的風格都有
我認為Wallpaper Engine最吸引人的地方就在於大部分的桌布都是「動態」的

「動態」指的是甚麼?
動態包含了讓桌布上的人物動起來，或是場景中的雲會飄動、雨會落下，還有像音訊回應這種炫砲的東西
這些都是由創作者手工製作並且調試的，有些桌布細膩的程度真的是令人嘆為觀止呢!

而為什麼Wallpaper Engine能有如此龐大的桌布數量?
其實Wallpaper Engine是一個平台，讓創作者能透過Steam獨特的「工作坊」上傳自己創作的獨一無二的桌布!
想讓自己的桌面變得更好看嗎? 趕快來試試Wallpaper Engine吧!

Steam頁面連結:
https://store.steampowered.com/app/431960/Wallpaper_Engine/

**Comment by Yunhans** [comment]
我的筆電沒有獨立顯卡會不會燒起來

**Reply by a45320614** [reply]
不用擔心這個問題，Wallpaper Engine提供了桌布的畫質設定，讓使用者能根據自己電腦的效能及需求調整，而「動態」桌布的部分確實比較吃顯卡的效能，不過也僅限於視窗在桌面的時候。當你瀏覽網頁或是做其他事情的時候，Wallpaper Engine會自動把「動態」桌布暫停，節省效能!

**Comment by danwan1027** [comment]
Mac 也可以使用這個軟體嗎

**Reply by a45320614** [reply]
Wallpaper Engine目前尚不支持Mac系統!
不過前幾個月Android已經開放下載了
算是對蘋果生態的使用者比較不友善~

**Reply by Chiway1229** [reply]
好可惜 我的蘋果手機裝不了

**Comment by Chiway1229** [comment]
我之前也在steam上買了這個套件
我覺得非常實用，不僅有著龐大的圖片、動圖庫可供選擇
還可以自己匯入圖片
肯定能找到自己喜歡的!

**Reply by a45320614** [reply]
確實，以在中央吃一頓飯就沒了的價格，這個軟體絕對是佛心來著!

**Comment by williamyen0623** [comment]
感覺會很好電（？
但動態桌面真香

**Comment by CPCpyt** [comment]
感謝分享~
我本人也有購買！常常匯入自己的相片
我覺得真的很棒，透過一個程式提升整個質感。
